images/runtime/python/3.11/bullseye.Dockerfile

```diff
@@ -49,11 +49,15 @@ RUN true
 RUN chmod +x /tmp/receiveGpgKeys.sh
 RUN chmod +x /tmp/build.sh
 
-RUN ${BUILD_DIR}/buildPythonSdkByVersion.sh $PYTHON_VERSION $DEBIAN_FLAVOR
+RUN --mount=type=secret,id=oryx_sdk_storage_account_access_token \
+    set -ex \
+    && export ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN="$(cat /run/secrets/oryx_sdk_storage_account_access_token)" \
+    && ${BUILD_DIR}/buildPythonSdkByVersion.sh $PYTHON_VERSION $DEBIAN_FLAVOR \
+    && export ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN=""
 
 RUN set -ex \
  && cd /opt/python/ \
- && ln -s 3.11.1 3.11 \
+ && ln -s 3.11.0 3.11 \
```

Comment:
Could you please clarify if you remember why this change from 3.11.1 to 3.11.0 was made?

Reply:
Dan, I did not make this change. Assuming it happened during a merge.